src/ui/DynamicFacetManager/DynamicFacetManager.ts

```diff
@@ -232,13 +232,26 @@ export class DynamicFacetManager extends Component {
 
     const [collapsableFacets, uncollapsableFacets] = partition(this.facetsWithDisplayedValues, facet => facet.options.enableCollapse);
     const [facetsWithActiveValues, remainingFacets] = partition(collapsableFacets, facet => facet.hasActiveValues);
-    const indexOfFirstFacetToCollapse =
+    let indexOfFirstFacetToCollapse =
       this.options.maximumNumberOfExpandedFacets - uncollapsableFacets.length - facetsWithActiveValues.length;
 
     facetsWithActiveValues.forEach(dynamicFacet => dynamicFacet.expand());
 
     remainingFacets.forEach((dynamicFacet, index) => {
-      index < indexOfFirstFacetToCollapse ? dynamicFacet.expand() : dynamicFacet.collapse();
+      if (index >= indexOfFirstFacetToCollapse) {
+        return dynamicFacet.collapse();
+      }
+
+      if (dynamicFacet.options.collapsedByDefault) {
+        dynamicFacet.logger.info(
```

Comment:
Is this actually a meaningful message ? 

If I was a end user of the framework and did not have prior knowledge of the way DNE works (with the manager and everything), then I'm not sure I would be able to understand why this information was displayed to me ?

Also, would you be able to explain here all the scenarios that would cause a facet to be expanded (or collapsed) by default ?

With the options on each facets, and the option on the manager, and what would happen for each combination ?

I'm not sure I actually understand/follow that part 100%.  
I'm wondering if they are all logical for a end user perspective.

Reply:
I'm not sure I understand this message either.

Does this imply that facets in a DynamicFacetManager should normally _not_ have their `collapsedByDefault` option set to `true`?

Reply:
I updated the msg after @olamothe 's comment

They should not have the collapsedByDefault option set to true without good reason. But if you do it you get an Info msg which is fair, it's not a warning

Reply:
I see :) Well, I was still a bit confused hehe

How about something along those lines:

> The facet has its "collapsedByDefault" option set to "true", which prevents the DynamicFacetManager from expanding it. While this configuration may be legitimate, it partially defeats the purpose of the dynamic navigation experience feature.

Reply:
Well, as new as I am to the framework, the message suggested by @fbeaudoincoveo made more sense to me. And maybe a More information link to 'Using Dynamic Facets' (https://docs.coveo.com/en/2917/javascript-search-framework/using-dynamic-facets)would be handy.